src/nyc_trees/apps/home/views.py

```diff
@@ -32,6 +35,28 @@ def retrieve_job_status(request):
     pass
 
 
+def redirect_to_flat_page(url):
+    def fn(request, *args, **kwargs):
+        return flatpage(request, url)
+    return fn
+
+
+def mark_user_on_success(attr):
```

Comment:
I like this as a wrapper. It looks like we are going to trigger "marks" not on the initial navigation to the page, but when the person clicks "Next" to move on to another section. This design has us covered. We can just adjust the "Next" button url and the redirect connected to the route.